lib/RuntimeGlobals.js

```diff
@@ -199,3 +199,9 @@ exports.amdOptions = "__webpack_require__.amdO";
  * the System polyfill object
  */
 exports.system = "__webpack_require__.System";
+
+/**
+ * the shorthand for Object.prototype.hasOwnProperty
+ * using of ot decreases the compiled bundle size
+ */
+exports.hasOwnProperty = "__webpack_require__.hop";
```

Comment:
Could you use a single letter property name to be even shorter. I think we used `__webpack_require__.o` before for this runtime function.